src/admin-views/editor/panel/ticket.php

```diff
@@ -152,7 +179,135 @@ class="tribe_soft_note ticket_form_right"
 						); ?>
 					</span>
 				</div>
-				<fieldset id="tribe_ticket_provider_wrapper" class="input_block" aria-hidden="true" >
+				<div class="input_block">
+					<label class="ticket_form_label ticket_form_left" for="ticket_start_date"><?php esc_html_e( 'Start sale:', 'event-tickets' ); ?></label>
+					<div class="ticket_form_right">
+						<input
+							autocomplete="off"
+							type="text"
+							class="tribe-datepicker tribe-field-start_date ticket_field"
+							name="ticket_start_date"
+							id="ticket_start_date"
+							value="<?php echo esc_attr( $ticket ? $start_date : null ); ?>"
+							data-validation-type="datepicker"
+							data-validation-is-less-or-equal-to="#ticket_end_date"
+							data-validation-error="<?php echo esc_attr( json_encode( $start_date_errors ) ) ?>"
```

Comment:
* :no_entry_sign: Inline PHP statement must end with a semicolon
* :warning: json_encode() is discouraged. Use wp_json_encode() instead.

<sub>hash: 0fd22e8a2c9f1113636ef1f8b1cd1908</sub>

Reply:
We should address both of these.

Reply:
done

Reply:
```suggestion
							data-validation-error="<?php echo esc_attr( wp_json_encode( $start_date_errors ) ); ?>"
```